spec/features/user_can_log_in_through_google_spec.rb

```diff
@@ -2,16 +2,12 @@
 
 RSpec.describe "User Can Sign In Through Google" do
   it "Sees that home Page with User name" do
-    
+
     stub_omniauth
 
     visit '/'
 
-    click_on "Login"
-
-    expect(current_path).to eq login_path
-
-    click_on "Sign In With Google"
+    click_on "Google"
```

Comment:
Nice refactor. Do we want it to just say "Google" or "Sign In with Google"?

Reply:
The google icon you guys made yesterday just says google so I left it at that. Doesnt matter to me though